packages/x-date-pickers-pro/src/DateRangeCalendar/useDragRange.ts

```diff
@@ -210,7 +210,7 @@ const useDragRangeEvents = <TDate>({
     setRangeDragDay(null);
     setIsDragging(false);
 
-    const target = resolveElementFromTouch(event, true);
+    const target = resolveElementFromTouch(event, false);
```

Comment:
As you can see, this change would introduce a regression of emitting an `onChange` when nothing has actually changed (you dragged around a day and dropped it where it is currently set).

Reply:
ah, yes ... I did not follow through `onDrop`. Will look into it. Thanks for pointing that out!